cmd/prom-aggregation-gateway/main_test.go

```diff
@@ -19,124 +19,124 @@ gauge 42
 counter 31
 # HELP histogram A histogram
 # TYPE histogram histogram
-histogram_bucket{le="1"} 0
-histogram_bucket{le="2"} 0
-histogram_bucket{le="3"} 3
-histogram_bucket{le="4"} 4
-histogram_bucket{le="5"} 4
-histogram_bucket{le="6"} 4
-histogram_bucket{le="7"} 4
-histogram_bucket{le="8"} 4
-histogram_bucket{le="9"} 4
-histogram_bucket{le="10"} 4
-histogram_bucket{le="+Inf"} 4
-histogram_sum{} 2.5
-histogram_count{} 1
+histogram_bucket{le="1"} 0  %[1]d
```

Comment:
All these changes to add `%[1]d` to metric output are mysterious to me.  Perhaps caused by an upstream change?
I prefer when all changes are explained by the commit message.